### PR TITLE
Added error handling to PDF sheet generation

### PR DESCRIPTION
**Pull request by agwosdz**
Added simple error handling to music sheet generation. 

Also aligned README with recent changes

Alex

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,15 +123,16 @@ _Not all options working now!_
     --crepe_step_size  unit is miliseconds >> ((default) is 10)
     
     [extra]
-    --hyphenation           True|False >> ((default) is True)
-    --disable_separation    True|False >> ((default) is False)
+    --hyphenation           Disable automatic hyphenation > True|False >> ((default) is True)
```
Comment (rakuri255) [inline README.md:126]: But we hyphen on default. So it should be:
```suggestion
    --hyphenation           Use automatic hyphenation > True|False >> ((default) is True)
```

Comment (rakuri255) [reply README.md:126]: or we change the flag to --disable_hyphenation ?

**Review by rakuri255 — CHANGES_REQUESTED** [review]

**Comment by agwosdz** [comment]
I have some QOL changes lined up somewhere that makes all the True/False parameters noargs

**Comment by agwosdz** [comment]
Also fixed the import of cyan_highlighted I missed, oops

**Review by rakuri255 — APPROVED** [review]

**Pull request merged**